### ISSUE TITLE
Vanilla-Extract: A Lightweight and Optimized Alternative to Tailwind for CSS Styling

### ISSUE THREAD

**Issue by meluiz** [joaom00/birth-social#1]
Tailwind is a widely used utility CSS class library for building user interfaces. While it is a great tool, it can be a bit heavy in terms of file size and can take more time to set up. This can impact your site's performance and ultimately affect the user experience.

On the other hand, Vanilla-Extract is a CSS style management tool that is extremely lightweight and highly optimized. It takes a different approach to creating styles, using JavaScript as its primary language and generating optimized CSS at the end of the process. As a result, Vanilla-Extract is extremely fast and easy to use, allowing you to create and manage your styles quickly and efficiently.

Additionally, Vanilla-Extract offers advanced features such as support for themes and light and dark modes, which are easy to implement and customize. These features can help improve the consistency and usability of your site, making it more attractive and user-friendly.

While there may be a learning curve when switching to Vanilla-Extract, in the long run, it can save time and significantly improve your site's performance. If you're looking for a lighter and more optimized alternative to Tailwind, Vanilla-Extract is definitely an option worth exploring.

**Comment by Caixetadev** [comment]
Discordo.

**Comment by revogabe** [comment]
mo preguiça

**Comment by Caixetadev** [comment]
Não se escreve "Tailwind" e sim "Tailwind CSS"

**Comment by meluiz** [comment]
> Não se escreve "Tailwind" e sim "Tailwind CSS"

You are correct, my apologies for the mistake. A more appropriate title would be "Vanilla-Extract: A Lightweight and Optimized Alternative to Tailwind CSS for CSS Styling". Thank you for bringing it to my attention!
